backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -615,6 +620,51 @@ public List<ApiUser> getUsersInCurrentOrg() {
     return usersInOrg;
   }
 
+  @AuthorizationConfiguration.RequirePermissionManageUsers
+  public Page<ApiUserWithStatus> getPagedUsersAndStatusInCurrentOrg(int pageNumber, int pageSize) {
+    Organization org = _orgService.getCurrentOrganization();
+
+    final Map<String, UserStatus> emailsToStatus =
+        _oktaRepo.getPagedUsersWithStatusForOrganization(org, pageNumber, pageSize);
+    List<ApiUser> users = _apiUserRepo.findAllByLoginEmailInOrderByName(emailsToStatus.keySet());
```

Comment:
This part of the method will be [updated in this future ticket](https://github.com/CDCgov/prime-simplereport/issues/8108) to fetch users from the DB instead of having to get them from Okta first.